articles/faq.md

```diff
@@ -65,30 +65,30 @@ All public data is queryable. For more information see [help on querying](/artic
-User certification ensures that you understand your responsibilities for sharing data through Synapse, especially data derived from human participants. These responsibilities include making sure that data derived from human participants is de-identified and that all applicable privacy laws and regulations are observed. For example, if working in the USA, data should be de-identified according to [HIPAA standards](http://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html). See the Data Use Procedures for more information.
+User certification ensures that you understand your responsibilities for sharing data through Synapse, especially data derived from human participants. These responsibilities include making sure that data derived from human participants is de-identified and that all applicable privacy laws and regulations are observed. See the [How to contribute data](link to how to contribute) for more information.
 
 #### How do I get certified?
-To become a certified user, you will need to pass a brief quiz that may accessed here.
+To become a certified user, you will need to pass a brief quiz that may accessed [here](link to quiz).
 
 #### Is everything I share on Synapse public?
-No. You can share your content with whomever you chose. By default, Projects and their content are visible only to the user who created it. You have the ability to grant other Synapse users, Synapse Teams, or the public access to your Project content.
+No. Use `Sharing settings` to control who can see the content you create. By default, `Projects` and their content are visible only to the user who created it. By using the Synapse `Sharing settings`, you have the ability to grant other Synapse users, Synapse teams, or the public access to your Project content. You can learn more here: [Sharing Settings and Conditions for Use](link here).
 
 #### Can I store sensitive information about human subjects in Synapse?
-Yes. Synapse has an IRB-approved data governance procedure that provides mechanisms to share sensitive data in a controlled manner. You can learn more by reading our Data Use Procedure documentation and [Governance documentation](https://www.synapse.org/#!Help:Governance). For further inquires or help in understanding how to apply these procedures to your data, please contact the Synapse Access and Compliance Team at [act@sagebase.org](mailto:act@sagebase.org).
+Yes. Synapse has an IRB-approved data governance procedure that employs `Conditions for Use` to allow for the sharing of sensitive data in a controlled manner. You can learn more by reading our [Sharing Settings and Conditions for Use](link here) documentation and [Governance documentation](https://www.synapse.org/#!Help:Governance). If you have questions or would like assistance in applying `Conditions of Use` to your data, please contact the Synapse Access and Compliance Team at [act@sagebase.org](mailto:act@sagebase.org).
 
```

Comment:
/articles/access_controls.html

and update Governance doc to:
/articles/governance.html